prow/plugins/bugzilla/bugzilla.go

```diff
@@ -337,7 +337,21 @@ Once a valid bug is referenced in the title of this pull request, request a bug
 				e.bugId, bc.Endpoint()))
 		}
 
-		valid, why := validateBug(*bug, options)
+		var dependents []bugzilla.Bug
+		if options.DependentBugStatuses != nil {
+			for _, id := range bug.DependsOn {
```

Comment:
Does bugzilla explicit prevent depending on yourself?  Some paranoia here might be a good idea (if id == bug, skip or report an error)

Reply:
>  You can't make a bug block itself or depend on itself. 

No, not possible

Reply:
![Screenshot from 2019-07-02 15-50-52](https://user-images.githubusercontent.com/7328370/60551754-30b12080-9ce1-11e9-97b9-6bec0337b151.png)